client/src/components/SignIn/index.js

```diff
@@ -196,7 +195,6 @@ class SignIn extends React.Component {
                         type='text'
                         className='validate lastname'
                         value={this.state.newLastName}
-                        placeholder='Last Name'
```

Comment:
don't like these placeholders?

Reply:
nope. get rid of the placeholders. they're redundant. it literally says the same thing right above the placeholder lol. We could get witty though and have it say something like "for better or for worse..." or "name your game squad (we don't care who your daddy is)", etc.

Reply:
agreed. my b missed those.